lib/config.js

```diff
@@ -82,6 +82,29 @@ function getEnv (prefix, name) {
   return process.env[envVar.toUpperCase().replace(/-/g, '_')]
 }
 
+/**
+ * Generate a secret based on {prefix}_SECRET_SEED.
+ *
+ * @param {String} prefix
+ * @param {String} name
+ * @returns {Buffer}
+ */
+function generateSecret (prefix, name) {
+  const seed = parseSecretSeed(prefix)
+  return crypto.createHmac('sha256', seed)
+    .update(name)
+    .digest()
+}
+
+function parseSecretSeed (prefix) {
+  const secretSeed = getEnv(prefix, 'SECRET_SEED')
```

Comment:
I'd just call it "secret", since "seed" is what our Ed25519 library calls its private key, so it's a bit confusing. Also, the equivalent config value in the ilp-kit API is `API_SECRET`, so it would make sense for this to be `LEDGER_SECRET`.

Since this is the main user-facing secret value and the main value a user will have to worry about, I like this idea of giving it a simple name.